distribution/docs/src/main/resources/content/_configuring/content-directory-monitor.adoc

```diff
@@ -68,23 +68,31 @@ For local files, the default value of 500 milliseconds is recommended. The recom
 It is also recommended that the *ReadLock Time Interval* be set to a lower amount of time when the *Maximum Concurrent Files* is set above 1 so that files are
 locked in a timely manner and processed as soon as possible.  When a higher *ReadLock Time Interval* is set, the time it takes for files to be processed is increased.
 
-.Attribute Mapper
-The CDM supports setting metacard attributes directly when ${branding} ingests a file. Custom mappings are entered in the form:
+.Attribute Overrides
+The CDM supports setting metacard attributes directly when ${branding} ingests a file. Custom overrides are entered in the form:
```

Comment:
Why is this documentation repeated? can one of these files be removed? @ricklarsen

Reply:
At first glance, it looks like it could be pared down. I'm not sure where it's of the most value, as it is repeated for (theoretically) two different audiences, but it essentially the same info. I'd say we should have it in one place or the other and link accordingly. 

This _seems_ to be configuration instructions, so I'd keep it there, but I don't know what, if anything, needs to be described in this architecture-overview section.

Reply:
I'd be in favor of keeping the configuring one and deleting the architecture one. 
I think it's a little bit much to have both.